src/main/java/org/commcare/core/network/CommCareNetworkServiceGenerator.java

```diff
@@ -39,13 +49,40 @@
         return response;
     };
 
+    private static Interceptor driftInterceptor = new Interceptor() {
+        @Override
+        public Response intercept(Chain chain) throws IOException {
+            Request request = chain.request();
+            Response response = chain.proceed(request);
+            ICommCarePreferenceManager commCarePreferenceManager = CommCarePreferenceManagerFactory.getCommCarePreferenceManager();
+            if (commCarePreferenceManager != null) {
+                String serverDate = response.header("date");
+                try {
+                    long serverTimeInMillis = new SimpleDateFormat("EEE, dd MMM yyyy hh:mm:ss zzz").parse(serverDate).getTime();
+                    long now = new Date().getTime();
+                    long currentDrift = (now - serverTimeInMillis) / HOUR_IN_MS;
```

Comment:
Just want to double check that this was an intended result- This means that we're counting a drift value of less than an hour as no drift. Is that what we want?

Reply:
Yeah. Drift is supposed to convey no. of hours and we are neglecting anything less than that.

Reply:
👍